controls/C-0256-exposuretointernet.json

```diff
@@ -20,10 +20,10 @@
             }
         ]
     },
-    "description": "",
-    "remediation": "",
-    "rulesNames": [],
-    "test": "",
+    "description": "fails if workloads are exposed on internet",
```

Comment:
Need to be more detailed on what exactly the control checks, and why (i.e. what is the risk).
See other "security" controls text examples.